packages/@lwc/ssr-compiler/src/compile-template/shared.ts

```diff
@@ -95,3 +103,38 @@ export function normalizeClassAttributeValue(value: string) {
     // @ts-expect-error weird indirection results in wrong overload being picked up
     return StringReplace.call(StringTrim.call(value), /\s+/g, ' ');
 }
+
+export function getChildAttrsOrProps(
```

Comment:
I moved this function to the shared file because I'm reusing it in `lwc-component.ts`.